aries_cloudagent/anoncreds/issuer.py

```diff
@@ -220,7 +222,7 @@ async def create_and_register_schema(
             raise AnonCredsIssuerError(
                 "Schema already exists but was not in wallet; stored in wallet"
             ) from err
-        except AnoncredsError as err:
+        except (AnoncredsError, BaseAnonCredsError, LedgerError) as err:
```

Comment:
`LedgerError` will only be raised by Indy components; `LedgerError` should be wrapped into a `BaseAnonCredsError` subtype rather than be handled in this abstract/general interface.

Reply:
Will do.

Reply:
updated related code